research/train_model.py

```diff
@@ -0,0 +1,11 @@
+import multiprocessing
+
+if __name__=="__main__":
+    model_name = "model.mdl"
+    model_input = "wiki.he.text"
```

Comment:
maybe allow adding these as arguments? 

same question for the previous script

(or at least save them in `consts.py`)

Reply:
f097ba782e8ed291798ff98478a061bfedf71e92